introduction.md

```diff
@@ -0,0 +1 @@
+I'm not good with words... so, it's just a file to atend the lesson.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
Electives